### ISSUE TITLE
"Internal Server Error" on login

### ISSUE THREAD

**Issue by ozone** [Aioros/destiny-reminders#27]
Here's how I can reproduce the problem:
1. visit https://destinyreminders.net/
2. click on the login icon in the top right of the screen, which redirects to bungie.net for approval
3. approve access for Destiny Reminders, get redirected back to destinyreminders.net, where all I see is "Internal Server Error".

I have no other indication of what causes the issue. You'll probably get an idea from the server logs (this happened 5-15min before I posted this). I can reproduce easily by visiting https://destinyreminders.net/auth/logout and following the above steps again.

I suspect this is due to my first account being a Stadia one (I also have a PSN one but it was added afterwards).

**Comment by Aioros** [comment]
Yep, my apologies, I inadvertently broke something with my previous commit. It should be fixed now, let me know if it works for you, and thanks for the heads up.
